ExtendedJson/ExtendedJson/Source/ExtendedJson/ExtendedJsonRepresentable.swift

```diff
@@ -218,6 +218,8 @@ extension ExtendedJsonRepresentable {
             return try String.fromExtendedJson(xjson: value!)
         case is Bool:
             return try Bool.fromExtendedJson(xjson: value!)
+        case let value as ExtendedJsonRepresentable:
+            return value
```

Comment:
This allows for the classes already coerced into `ExtendedJsonRepresentable` to be added to the Document without issue

Reply:
nice. worth adding a test for this case?